packages/cdktf/lib/terraform-backend.ts

```diff
@@ -0,0 +1,26 @@
+import { Construct } from "constructs";
+import { TerraformElement } from "./terraform-element";
+
+export abstract class TerraformBackend extends TerraformElement {
+    constructor(scope: Construct, id: string, protected readonly name: string) {
+        super(scope, id);
+    }
+
+    // jsii can't handle abstract classes?
+    protected synthesizeAttributes(): { [name: string]: any } {
+        return {}
+    }
```

Comment:
I'm wondering if it'd make sense to support escape hatches here as well?

Reply:
It's still possible to used the stack level escape hatch, but I can try adding support.